### ISSUE TITLE
bower install vs. bower install --save

### ISSUE THREAD

**Issue by intcreator** [cheonhyangzhang/paper-tags-input#16]
The installation instructions have the following command: 

```
bower install paper-tags-input
```

However, this only adds the needed files and does not add the element to the bower.json.  The command should be updated as follows:

```
bower install --save paper-tags-input
```

This will add the element to the bower.json so it can be installed with `bower install`.  This will also make it more consistent with other Polymer element documentation (see [`<paper-button>`](https://elements.polymer-project.org/elements/paper-button) for example).

**Comment by cheonhyangzhang** [comment]
Sure. Makes sense. will change to --save

**Comment by cheonhyangzhang** [comment]
This issue is handled by pull request #17

**Comment by cheonhyangzhang** [comment]
@intcreator 

The changes are implemented in v1.2.4
https://github.com/cheonhyangzhang/paper-tags-input/releases/tag/v1.2.4
